google/cloud/ndb/model.py

```diff
@@ -4323,6 +4323,40 @@ def _prepare_for_put(self, entity):
             if value is not None:
                 value._prepare_for_put()
 
+    def _to_datastore(self, entity, data, prefix="", repeated=False):
+        """Override of :method:`Property._to_datastore`.
+
+        Although this property's entities should be stored as serialized
+        strings, when stored using old NDB they appear as unserialized
+        entities in the datastore. When serialized as strings in this class,
+        they can't be read by old NDB either. To avoid these incompatibilities,
+        we store them as entities when legacy_data is set to True, which is the
+        default behavior.
+        """
+        # Avoid Python 2.7 circularf import
```

Comment:
typo here

Reply:
That typo had been copy/pasted in several places. Fixed.